xonsh/aliases.py

```diff
@@ -73,6 +73,19 @@ def source_bash(args, stdin=None):
     return
 
 
+def exec(args, stdin=None):
```

Comment:
This name has to be something different because it is overriding the Python `exec()` builtin.  This isn't a bash exec, either.  Note that `execx()` is taken already by the xonsh builtin function.  I am not sure that this is *that* much different than `execx()` in spirit, either.  Maybe this should be a thin wrapper around that.